src/bindings/c/src/ov_core.cpp

```diff
@@ -17,6 +17,24 @@ char* str_to_char_array(const std::string& str) {
     return char_array;
 }
 
+static char* last_err_msg = nullptr;
```

Comment:
is it possible to store `last_err_msg` as `std::string` and convert to `char*` directly in `ov_get_last_err_msg()`?

Reply:
It seems the string contents will be lost if store exception.what() into std::string variable:
      `static std::string last_err_msg ;
       last_err_msg = std::string(ex.what())`

Maybe there is some mistake why the content has been lost in std::string.

Reply:
Fixed the content lost issue and use std::string, thank your suggestion! @vurusovs